filters/addbackground.go

```diff
@@ -58,12 +70,11 @@ func (r *addBackground) CreateFilter(args []interface{}) (filters.Filter, error)
 		return nil, err
 	}
 
-
 	return f, nil
 }
 
-func (r *addBackground) Request(ctx filters.FilterContext) {}
+func (s *addBackground) Request(ctx filters.FilterContext) {}
 
-func (r *addBackground) Response(ctx filters.FilterContext) {
-	HandleImageResponse(ctx, r)
+func (s *addBackground) Response(ctx filters.FilterContext) {
+	HandleImageResponse(ctx, s)
```

Comment:
All the filters call HandleImageResponse https://github.com/zalando-incubator/skrop/blob/merge-filters/filters/cropbywidth.go#L83